reader.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func getFilesFromFolder(folder string) (files []string, err error) {
+	err = filepath.Walk(folder,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if info.IsDir() == false {
+				files = append(files, path)
```

Comment:
Onde essa variável está sendo declarada?

Reply:
No retorno do método. É uma má prática usar assim?

Reply:
Realmente. Passei batido. 

Não chamaria de má prática, porém nesse caso, eu sigo essa visão:

> 
Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values. Corollary: it's not worth it to name result parameters just because it enables you to use naked returns. Clarity of docs is always more important than saving a line or two in your function.

Fonte: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
Boa, valeu por explicar!